apps/site/package.json

```diff
@@ -17,12 +17,15 @@
 		"@radix-ui/react-select": "^2.0.0",
 		"@radix-ui/react-tabs": "^1.0.4",
 		"@radix-ui/react-toast": "^1.1.5",
+		"@radix-ui/themes": "^2.0.3",
 		"@react-three/drei": "^9.88.14",
 		"@react-three/fiber": "^8.15.11",
 		"@sanity/image-url": "^1.0.2",
 		"@types/three": "^0.158.2",
 		"axios": "^1.6.2",
 		"clsx": "^2.0.0",
+		"date-fns-tz": "^2.0.0",
+		"dayjs": "^1.11.10",
```

Comment:
Thought/Suggestion: Day.js provides their own [Timezone plugin](https://day.js.org/docs/en/plugin/timezone), so is there a particular advantage in using this other date-fns-tz library?

Reply:
I wasn't able to get a Date object in PST using Day.js. From what I've tried, Day.js was able to print the date in PST but when the dayjs object gets converted into a JS Date object, the Date object is still in UTC time.

Reply:
Will revisit date processing.